src/SamplesApp/SamplesApp.Shared/App.Assertions.cs

```diff
@@ -203,4 +210,12 @@ private void AssertIssue10313ResumingAfterActivate()
 			Assert.Fail("Resuming should never be triggered unless the app is suspended.");
 		}
 	}
+
+	private static void AssertWindowSize(WindowSizeChangedEventArgs e)
+	{
+		// The idea is just to make sure the bounds are set correctly before WindowSize event is fired.
+		var applicationViewBounds = ApplicationView.GetForCurrentView().VisibleBounds;
```

Comment:
Does this work on WinUI? I think the API just throws 🤔

Reply:
@MartinZikmund Yes, it throws on WinUI. See https://github.com/unoplatform/uno/pull/14950#discussion_r1440244473

Reply:
We have *internal* usages of `ApplicationView.GetForCurrentView()` which is why we care about VisibleBounds being correctly set when in SizeChanged event.

Reply:
Though this would fail in the winui sample, right?

Reply:
Yup. I wrapped the problematic call under `HAS_UNO` in the last commit.